Dev.to organization comments github action is failing

We've had the dev.to Orbit integration running fine for a while now, but it just started throwing this error a couple days ago if anyone has any ideas!

```
Run dev_orbit --check-organization-comments
/opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/dev_orbit-0.5.1/lib/dev_orbit/dev.rb:86:in `get_articles': undefined method `empty?' for #<Net::HTTPForbidden 403 Forbidden Bots readbody=true> (NoMethodError)
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/dev_orbit-0.5.1/lib/dev_orbit/dev.rb:20:in `process_comments'
Checking for new DEV organization comments and posting them to your Orbit workspace...
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/dev_orbit-0.5.1/lib/dev_orbit/client.rb:72:in `organization_comments'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/dev_orbit-0.5.1/scripts/check_org_comments.rb:13:in `render'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/base.rb:485:in `start'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/dev_orbit-0.5.1/bin/dev_orbit:54:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/bin/dev_orbit:23:in `load'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/bin/dev_orbit:23:in `<main>'
Error: Process completed with exit code 1.
```

--- a/lib/dev_orbit/dev.rb
+++ b/lib/dev_orbit/dev.rb
@@ -76,6 +76,7 @@ module DevOrbit
         https.use_ssl = true
 
         request = Net::HTTP::Get.new(url)
+        request["User-Agent"] = "community-ruby-dev-orbit/#{DevOrbit::VERSION}"
 
         response = https.request(request)
 
@@ -105,6 +106,7 @@ module DevOrbit
 
         request = Net::HTTP::Get.new(url)
         request["api_key"] = @api_key
+        request["User-Agent"] = "community-ruby-dev-orbit/#{DevOrbit::VERSION}"
 
         response = https.request(request)
 
@@ -127,6 +129,7 @@ module DevOrbit
       https.use_ssl = true
 
       request = Net::HTTP::Get.new(url)
+      request["User-Agent"] = "community-ruby-dev-orbit/#{DevOrbit::VERSION}"
 
       response = https.request(request)
 

